content/automate/ManageIQ/System/Event/EmsEvent/RHEVM.class/importexport_import_vm_failed.yaml

```diff
@@ -10,3 +10,5 @@ object:
   fields:
   - meth4:
       value: update_vm_import_status
+  - rel5:
+      value: "/System/event_handlers/event_action_refresh?target=src_vm"
```

Comment:
@smelamud @agrare 
When the VM import fails do we have to remove the VM from our database? Should that be a different kind of action_refresh?